### ISSUE TITLE
graphframes-0.8.3-spark3.3-s_2.12.jar is missing

### ISSUE THREAD

**Issue by pbauwens** [graphframes/graphframes#440]
According to the code base there should be a version available for Spark 3.3, 3.4 and 3.5

While 3.4 and 3.5 versions are there, 3.3 is missing in action

**Comment by kagesenshi** [comment]
looks missing here https://spark-packages.org/package/graphframes/graphframes

so attempting to install using `--packages graphframes:graphframes:0.8.3-spark3.3-s2.12` did not work ..

**Comment by praetp** [comment]
Any update on this

**Comment by rjurney** [comment]
I will check on this as part of my build changes. @praetp the preferred way to use the project is via the argument @kagesenshi gave.

**Comment by rjurney** [comment]
You're right, 3.3 is missing. I'll upload the next build, 0.8.5 with these versions when I ship it in a week or so.

https://spark-packages.org/package/graphframes/graphframes
